html/index.html

```diff
@@ -0,0 +1,223 @@
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <header>
+      <div class="nav-container">
+        <div class="desktop-logo">
+          <a href="index.html"
+            ><img src="/assets/cic_logoReal.png" alt="logo"
+          /></a>
+          <div class="hamburger nav-toggle" id="menu">
+            <i class="fas fa-bars fa-3x"></i>
+          </div>
+          <div class="close-menu" id="close">
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="webpages-menu" id="mob-nav">
+          <ul class="menu-controls">
+            <li class="webpages">
+              <a href="index.html">Home</a>
+            </li>
+            <li class="webpages">
+              <a href="about.html">About</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Program</a>
+            </li>
+            <li class="webpages">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="webpages">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company-name">
+            <a href="index.html">CIC Campaign</a>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <section class="first-section">
+      <div class="first-section-container">
+        <h2 class="open-intro-heading">"Hello! Connect, Learn & Grow"</h2>
+        <h1 class="open-heading">
+          ceos international community global summit 2021
+        </h1>
+        <div class="first-section-para">
+          <p>
+            A three-day event to learn from the best in their respective
+            Industries! CEOs Internatioal Community Global Summit'21 brings
+            together successful entrepreneurs, venture capitalists, angel
+            investors and advisors to teach you the tricks of the trade, offer
+            advice, tips and tools for aspiring entrepreneurs as they launch
+            their business.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3><i class="far fa-calendar-alt"></i> 2021.10.4(MON) ~ 6(WED)</h3>
+          <p><i class="far fa-clock"></i> 19:00 ~ 06:00 (UTC+1)</p>
+          <p class="venue">
+            <i class="fas fa-map-marker-alt"></i> M.K.O Abiola National Stadium
+            Abuja, Nigeria
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="second-section">
+      <div class="second-section-container">
+        <h2 class="second-section-heading">Main Program</h2>
+        <hr />
+        <div class="program-list">
+          <div class="lecture flex-box">
+            <img class="icons" src="/assets/program1.png" alt="program-icon" />
+            <h2>Lecture</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
```

Comment:
- [ ] The whole section in small mobile views of up to 550px, the information is squeezed. Kindly add margins and fix the height of the images ⬇️ 

![bn](https://user-images.githubusercontent.com/33067256/129426050-25757b74-68ac-4fbd-921b-2df1147b9f77.png)

- [ ] From html semantics, the whole section should be wrapped in an `article` tag